### DISCUSSION TITLE
Flutter walletconnect error

### DISCUSSION THREAD

**Discussion by Ririchy81** [WalletConnect/walletconnect-monorepo#2263] [Developer Support]
Hello , I'm have created an android application front-end in flutter and Blockchain as a backend but while Integration I'm facing an issue while connecting the Wallet, as a result it shows "Unsupported chain Id "
the application supports bep20 smartchain (bsc smartchain)

**Comment by finessevanes** [comment]
Hey - can you include the implementation as well ask the SDK information?

**Comment by finessevanes** [comment]
Closing this discussion due to three weeks of inactivity.
